api/src/pcapi/core/users/factories.py

```diff
@@ -415,6 +415,17 @@ def _build(
         return instance
 
 
+class AnonymizedUserFactory(UserFactory):
+    roles = [models.UserRole.ANONYMIZED]
+    email = factory.Sequence("anonymous_{}@anonymized.passculture".format)
+    address = factory.Sequence("{} place des noces rouges".format)
+    city = "La Rochelle"
```

Comment:
Pour plus de réalisme, `address`, `city` et `postalCode` sont à `None` pour un utilisateur anonymisé. Ça ne change rien pour ce cas mais si on utilise cette _factory_ plus tard pour autre chose, ça peut servir.

Rien ne t'empêche cependant de mettre l'iris d'un endroit que tu aimes bien :)

```suggestion
    address = None
    city = None
    postalCode = None
```

Reply:
Il n'y a pas forcement d'iris et en créer a la chaine est compliqué puisque les shape ne doivent pas se superposer et que les codes doivent être uniques

Reply:
C'est plutôt simple, je l'ai fait dans la sandbox : 
https://github.com/pass-culture/pass-culture-main/blob/v269.0.1/api/src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_gdpr_users.py#L185

Reply:
Oui j'avais trouvé ce code mais la tu as mis une shape en dure, faire des shape dynamiques qui ne se superposent pas, c'est ça qui me pose probleme.

Reply:
J'imaginais simplement donner le même iris à tous les utilisateurs anonymes. Mais de toute façon c'est mergé et on n'en a pas besoin pour ce ticket ; on verra au moment où ça servira :)